### PR TITLE
getting started to contribute with. Add makefile commands and improve CONTRIBUTING.MD

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
- https://github.com/kubernetes-sigs/kubebuilder/issues/978

## What 
Small changes in order to make clear the fixed path that the devs should use as add script to test the build and e2e in the makefile.

### CODE REVIEW

**Comment by camilamacedo86** [comment]
c/c @DirectXMan12

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,15 +15,9 @@ Please see https://git.k8s.io/community/CLA.md for more info.
...
-1. Build
-```sh
-$ make build
-```
-
-1. Test
-```sh
-$ ./test.sh
-```
+1. Ensure that the kubebuilder is cloned in the path `$GO_PATH/src/sigs.k8s.io/kubebuilder`
```
Comment (DirectXMan12) [inline CONTRIBUTING.md]: This should not be necessary, since KB uses Go modules.  If it is, we've got a bug in our build scripts.

File: Makefile [hunk 2]
```diff
@@ -34,3 +34,8 @@ generate:
 test:
 	go test -v ./cmd/... ./pkg/...
 
+test/build:
```
Comment (DirectXMan12) [inline Makefile]: These are phony targets and need to be marked as such.

Separately: nit-wise, I'm not a huge fan of the name of this -- maybe `test-project-generation`?

File: CONTRIBUTING.md (hunk 1 above)
Comment (DirectXMan12) [reply CONTRIBUTING.md]: (as long as you start with `GO111MODULE=on`)

Comment (camilamacedo86) [reply CONTRIBUTING.md]: you probably right.

File: Makefile (hunk 2 above)
Comment (camilamacedo86) [reply Makefile]: Done 👍

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -15,15 +15,9 @@ Please see https://git.k8s.io/community/CLA.md for more info.
...
-1. Build
-```sh
-$ make build
-```
-
-1. Test
-```sh
-$ ./test.sh
-```
+1. Ensure you activate module support before continue (`$export GO111MODULE=on`)
```
Comment (DirectXMan12) [inline CONTRIBUTING.md]: nit:

```suggestion
1. Ensure you activate module support before continue (`$ export GO111MODULE=on`)
```

Comment (camilamacedo86) [reply CONTRIBUTING.md]: 👍

**Comment by camilamacedo86** [comment]
HI @DirectXMan12,

All requests are done. Please, feel free to check. really tks for your time.

**Comment by DirectXMan12** [comment]
thanks!

**Pull request merged**